src/.nuget/Microsoft.NETCore.Jit/win/Microsoft.NETCore.Jit.pkgproj

```diff
@@ -9,9 +9,13 @@
   </PropertyGroup>
   <ItemGroup>
     <ArchitectureSpecificNativeFile Include="$(BinDir)clrjit.dll" />
+    <CrossArchitectureSpecificNativeFile Include="$(BinDir)$(CrossTargetComponentFolder)\clrjit.dll" />
     <File Include="@(ArchitectureSpecificNativeFile)">
       <TargetPath>runtimes/$(PackageTargetRuntime)/native</TargetPath>
     </File>
+    <File Condition="'$(HasCrossTargetComponents)' == 'true'" Include="@(CrossArchitectureSpecificNativeFile)">
+      <TargetPath>runtimes/$(CrossTargetComponentFolder)_$(PackagePlatform)/native</TargetPath>
```

Comment:
Not sure if this is correct. In coreclr package, cross target components are placed in tools/$(CrossTargetComponentFolder)_$(PackagePlatform) folder because they are not required at runtime. @gkhanna79 can you take a look?